### PR TITLE
[Workflow] Added workflow_transition_blockers twig function

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | https://github.com/symfony/symfony-docs/pull/11268
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/11268

EUFOSSA

---

also related to https://github.com/symfony/symfony/issues/26689

I'm not a big fan of the current name. What should I pick?

cc @javiereguiluz

### CODE REVIEW

File: src/Symfony/Bridge/Twig/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * added the `form_parent()` function that allows to reliably retrieve the parent form in Twig templates
+ * added the `workflow_build_transition_blocker_list()` function
```
Comment (HeahDude) [inline src/Symfony/Bridge/Twig/CHANGELOG.md]: `workflow_get_transition_blockers`?

Comment (lyrixx) [reply src/Symfony/Bridge/Twig/CHANGELOG.md]: in PHP, the method is named `build` (not `get`). I prefer to use the same vocabulary.

Did you see @javiereguiluz suggestion? I like them a lot

Comment (OskarStark) [reply src/Symfony/Bridge/Twig/CHANGELOG.md]: workflow_transition_blockers ?

Comment (lyrixx) [reply src/Symfony/Bridge/Twig/CHANGELOG.md]: Let's go with your proposal. I updated the PR. Thanks

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**